.env.template

```
@@ -9,8 +9,8 @@ OPENCV_VERSION=4.12.0
 DISCORD_TOKEN=''
 SEAWEED_SECRET_KEY=''
 DASHBOARD_ADMIN_USERNAME=
-DASHBOARD_ADMIN_PASSWORD=''
-DASHBOARD_ADMIN_DISCORD_ID=
-DASHBOARD_ORIGIN=http://localhost:5000
+DASHBOARD_ORIGIN=http://localhost:5173
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DASHBOARD_ORIGIN key should go before the DISCORD_TOKEN key

(UnorderedKey)